modules/datadog/widget.go

```diff
@@ -24,31 +23,26 @@ const HelpText = `
 
 type Widget struct {
 	wtf.HelpfulWidget
-	wtf.TextWidget
 	wtf.KeyboardWidget
+	wtf.ScrollableWidget
 
-	app      *tview.Application
```

Comment:
Note that this gets rid of most of the rest of app in widgets themselves. There are still a few outliers, for widgets with more complexity.